src/core/Akka.Tests/Pattern/CircuitBreakerSpec.cs

```diff
@@ -85,7 +85,8 @@ public async Task Must_increment_failure_count_on_callTimeout_before_call_finish
             var breaker = ShortCallTimeoutCb();
 #pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
             // meant to run as detached task
-            Task.Run(() => breaker.Instance.WithSyncCircuitBreaker(() => Thread.Sleep(Dilated(TimeSpan.FromSeconds(1)))));
+            var t = Task.Run(() => breaker.Instance.WithSyncCircuitBreaker(() => Thread.Sleep(Dilated(TimeSpan.FromSeconds(1)))));
+            await AwaitConditionAsync(() => t.Status >= TaskStatus.Running); // need to kick off the task before we can check the latch
```

Comment:
This spec is pretty racy due to non-determinism of the `TaskScheduler` - now we force our assertions to wait until the `Task.Run` actually gets scheduled for execution, because otherwise the `breaker` never gets a chance to be scheduled and this will throw the timing off.